.github/workflows/ruby.yml

```diff
@@ -6,23 +6,39 @@
 # For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
 
 name: Ruby
-on: [push, pull_request]
+on: [push]
 jobs:
+  check_react_and_ujs:
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby: [2.7]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler: 2.4.9
+          ruby-version: ${{ matrix.ruby }}
+      - uses: actions/setup-node@v3
+      - run: bundle install
+      - run: yarn
+      - run: bundle exec rake react:update
+      - run: bundle exec rake ujs:update
+      - run: ./check_for_uncommitted_files.sh
```

Comment:
Maybe we can follow the DRY principle and create a rake task for this check. We can use it here and also in the release task.